tests/test_diffeq/test_ode/test_ivp_examples.py

```diff
@@ -157,21 +157,27 @@ class TestJacobianEvaluation(unittest.TestCase, NumpyAssertions):
     def setUp(self):
         self.tspan = (0.0, 4.212)
 
+        self.dt = 1e-6  # should lead to errors ~1e-8 in the present tests
+        self.rtol = 1e-4
+
     def test_logistic_jacobian(self):
         rv = Constant(0.1)
         lg1 = ivp_examples.logistic(self.tspan, rv)
         random_direction = 1 + 0.1 * np.random.rand(lg1.dimension)
         random_point = 1 + np.random.rand(lg1.dimension)
         fd_approx = (
             0.5
-            * 1e11
+            * 1.0
```

Comment:
The `* 1.0` can be removed, do you want to leave it in for didactical reasons?

Reply:
yeah kinda (I think it makes it easier that this is the denominator of the fraction), though I am not too set on this opinion.